sprite.go

```diff
@@ -509,6 +509,14 @@ func (p *Sprite) Destroy() {
 	}
 }
 
+func (p *Sprite) DeleteThisClone() {
```

Comment:
It is advisable to comment for the purpose of `DeleteThisClone` here, as it appears redundant with `Destroy` and `Die` provided.